pkg/apis/servicecatalog/types.go

```diff
@@ -202,30 +202,30 @@ const (
 	ConditionUnknown ConditionStatus = "Unknown"
 )
 
-// ServiceClassList is a list of ServiceClasses.
-type ServiceClassList struct {
+// ClusterServiceClassList is a list of ClusterServiceClasses.
+type ClusterServiceClassList struct {
 	metav1.TypeMeta
 	metav1.ListMeta
 
-	Items []ServiceClass
+	Items []ClusterServiceClass
 }
 
 // +genclient=true
 // +nonNamespaced=true
 
-// ServiceClass represents an offering in the service catalog.
-type ServiceClass struct {
+// ClusterServiceClass represents an offering in the service catalog.
+type ClusterServiceClass struct {
 	metav1.TypeMeta
 	metav1.ObjectMeta
 
-	Spec   ServiceClassSpec
+	Spec   ClusterServiceClassSpec
 	Status ServiceClassStatus
```

Comment:
Status needs to be renamed

Reply:
We left the BrokerStatus as was as I was not thinking the status would necessarily be specific to Cluster level resource. So for consistency, didn't rename it here either.